my-code/src/containers/Home.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from 'react'
+import SearchInput from '../components/searchInput';
+import MovieCard from '../components/movieCard';
+import logo from '../assets/images/logo.svg';
+import { withRouter } from 'react-router-dom';
+import EmptyStateManager from '../components/emptyStateManager';
+
+class Home extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      data: [],
+      searchValue: '',
+      loading: false,
+      error: '',
+      addedMovies: [],
+    }
+  }
+
+  componentDidMount() {
+    if(localStorage.getItem('addedMovies')) {
```

Comment:
Same static values like `addedMovies` in constant above, it's a great practice to know that this value will not change and to share between functions and scopes.

Read more: https://www.robinwieruch.de/javascript-naming-conventions